src/core/context_processors.py

```diff
@@ -1,15 +1,25 @@
 from django.conf import settings
+from django.utils.translation import get_language
 
 from sponsors.models import Sponsor
 
 
+def _build_google_form_url(uid):
+    return 'https://docs.google.com/forms/d/e/{uid}/viewform'.format(uid=uid)
+
+
 def pycontw(request):
+    lang = get_language()
+    if lang and lang.startswith('zh'):
+        sponsor_id = '1FAIpQLSf5pgDeYtspU1AfexFlAw-gZXWjcXhPxNTj7HGb258bD-0Eig'
+    else:
+        sponsor_id = '1FAIpQLSeB7I99Jugc9qGqzxZfF_sYQTz3nE7--NDu0hHhoBpYSqrtyg'
     return {
         'GA_TRACK_ID': settings.GA_TRACK_ID,
         'KKTIX_URL': 'https://pycontw.kktix.cc/events/pycontw2017',
-        'VOLUNTEER_FORM_URL': (
-            'https://docs.google.com/forms/d/e/'
-            '1FAIpQLSfnTEpD_I74Yeji6GqfI-YSZoYgg7Ax-YCEG5PNHrBusTgn4Q/viewform'
+        'SPONSOR_FORM_URL': _build_google_form_url(sponsor_id),
+        'VOLUNTEER_FORM_URL': _build_google_form_url(
+            '1FAIpQLSfnTEpD_I74Yeji6GqfI-YSZoYgg7Ax-YCEG5PNHrBusTgn4Q',
         ),
     }
```

Comment:
Updated version of urls of sponsorship form:
```
2018 贊助商申請表單（中文）
https://goo.gl/P3zqmL

2018 贊助商申請表單（英文）
https://goo.gl/MLZTWU
```

`def _build_google_form_url` can be gone then. Not sure they worth a def.

Reply:
There’s still the volunteer form. But I can create a short URL for it too.